src/host/PtySignalInputThread.cpp

```diff
@@ -170,7 +170,6 @@ bool PtySignalInputThread::_GetData(_Out_writes_bytes_(cbBuffer) void* const pBu
         if (lastError == ERROR_BROKEN_PIPE)
         {
             _Shutdown();
-            return false;

```

Comment:
(okay sorry to nitpick, but all this code is terribly fragile)

why are we removing these `return false`s in `_GetData`? Aren't these needed to get the input threads to break out of their while loops?

Reply:
Oh right, cause we'll `ExitProcess` in `RundownAndExit` in `_Shutdown`, that makes sense.

Reply:
So here's the story behind this change:
Initially I tried to add some code after `ServiceLocator::RundownAndExit(ERROR_BROKEN_PIPE);`. This doesn't make any sense though, as `RundownAndExit` never returns, but neither the regular compilation nor the AuditMode marked this as an issue. If we use the `[[noreturn]]` annotation however we can turn this programming error into an immediate compilation warning. This is why this PR adds `[[noreturn]]`.

But now that code after calling a `[[noreturn]]` function is a warning I was forced to remove such unreachable code. This is why this PR removes the above line (and the other one).

Do you agree with this change? If not, would you like me to revert it?

Reply:
oh yea no, this makes sense, more sense than before. Defs keep this.